apps/io-services-cms-webapp/env.example

```diff
@@ -32,4 +32,14 @@ AZURE_APIM=
 AZURE_APIM_RESOURCE_GROUP= 
 AZURE_SUBSCRIPTION_ID=
 
-AzureWebJobsStorage=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://fnstorage:10000/devstoreaccount1;QueueEndpoint=http://fnstorage:10001/devstoreaccount1;TableEndpoint=http://fnstorage:10002/devstoreaccount1;
```

Comment:
Why this parameter has been removed?

Reply:
There is no reference in either the `config.ts` or the `webapp.tf` file, so I assumed it was an unused value.
I think it is a configuration for the old version of the _tf_ module... @balanza can you confirm that?

Reply:
It's a mandatory value as it's expected by Azure Function ([docs](https://learn.microsoft.com/en-my/azure/azure-functions/functions-app-settings)).

We can decide to enforce validation by adding it to `config.ts`. We don't always do it as we take the value for granted. I think the app won't even start without it (I don't remember right now)

Reply:
OK...so, the fact that it is not in the `webapp.tf` file (as an env var) is an error, right?